ci/submit_jobs_WCOSS2.sh

```diff
@@ -24,5 +24,15 @@ if [[ "$have_ifi" == "yes" && "$disable_ifi" == "no" ]] ; then
   done
 fi
 
+# Run additional GTG tests
+if [[ "$have_gtg" == "yes" && "$disable_gtg" == "no" ]] ; then
```

Comment:
```suggestion
if [[ "$have_ifi" == "yes" && "$disable_ifi" == "no" && "$have_gtg" == "yes" && "$disable_gtg" == "no" ]] ; then
```
The dafs test includes both ifi and gtg generation.